spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewriteDataFilesAction.java

```diff
@@ -902,7 +902,7 @@ public void testSortCustomSortOrder() {
   @Test
   public void testSortCustomSortOrderRequiresRepartition() {
     Table table = createTable();
-    writeRecords(20, SCALE, 20);
+    writeRecords(20, SCALE, 4);
```

Comment:
Nit: can we make variable partitions = 4?  As it's used twice , and for self-documentation.